### ISSUE TITLE
Scale order indicators on low dpi

### ISSUE THREAD

**Issue by guilledk** [pikers/piker#248]
Seems the color order/alert indicators aren't scaled depending on dpi, example: https://i.imgur.com/iFchFZf.png

**Comment by guilledk** [comment]
Also need to scale profit and loss report

**Comment by goodboy** [comment]
Yeah, this would make sense since I don't think we have dpi scaling around the paths used to draw all the annotation geos..

A lot of this was pulled from code inside `pyqtgraph` so we might need to do some scaling tweaks:
- here is the path geometry [helper](https://github.com/pikers/piker/blob/master/piker/ui/_annotate.py#L32)
- and *i think* [this is how we scale it?](https://github.com/pikers/piker/blob/master/piker/ui/_annotate.py#L101)

Probably worth some tweaking and testing by someone with lodpi 😉
